File: 09. Array/B. Balik Daftar/b_balik_daftar.java
import java.util.Scanner;

class b_balik_daftar {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int idx = 0;
        int[] arr = new int[101];
        while (scan.hasNextInt()) {
            arr[idx] = scan.nextInt();
            idx++;
        }
        scan.close();
        while (idx > 0) {
            idx--;
            System.out.println(arr[idx]);
        }
    }
}

File: 09. Array/B. Balik Daftar/b_balik_daftar.go
package main

import(
    "fmt"
)
func main(){
    var arr[101]int
    idx := 0
    for{
        _, err := fmt.Scan(&arr[idx])
        if (err != nil){
            break
        }
        idx += 1
    }
    for i := idx-1; i >= 0; i--{
        fmt.Println(arr[i])
    }
}
